src/tls13.c

```diff
@@ -5710,7 +5717,7 @@ static void RefineSuites(WOLFSSL* ssl, Suites* peerSuites)
     if (AllocateSuites(ssl) != 0)
         return;
 
-    XMEMSET(suites, 0, WOLFSSL_MAX_SUITE_SZ);
+    XMEMSET(suites, 0, sizeof(suites));
```

Comment:
Safer (future proof), because it is always going to be the right size of the object, whereas if suites is created with a size different than WOLFSSL_MAX_SUITE_SZ then there's a problem.